proofs/alf/cvc5/theories/BitVectors.smt3

```diff
@@ -304,9 +299,21 @@
 
 (declare-const BITVECTOR_EAGER_ATOM (-> Bool Bool))
 
+; internal operators
+
+(declare-const @bitOf
+    (-> (! Int :var m :implicit)
+        Int (BitVec m) Bool))
+
+(declare-const @bbT (->
+  (! Int :var n :implicit)
+  Bool
+  (BitVec n)
+  (BitVec (alf.add 1 n))) :right-assoc-nil @bvempty
+)
+
 ; symbolic constant
-;(declare-const bv (-> Int (! Int :var w) (BitVec w)))
+(declare-const @bv (-> Int (! Int :var w) (BitVec w)))
```

Comment:
How is this different from an uninterpreted constant?

Reply:
For now, it is essentially an uninterpreted function.  However, we will add an evaluator for it that transforms @bv to a binary constant.